python/metrics/src/hydrotools/metrics/metrics.py

```diff
@@ -44,7 +44,7 @@ def mean_error(
     power: float = 1.0,
     root: bool = False
     ) -> float:
-    """Compute the mean error or deviation. Default is Mean Absolute Error. The mean error 
+    r"""Compute the mean error or deviation. Default is Mean Absolute Error. The mean error 
```

Comment:
This is the first time i've seen a docstring defined as a raw string literal. I assume this avoids character escaping headaches in the latex expressions?

Reply:
Yeah, the expressions were raising DeprecationWarnings and this was the easiest solution.